docker/rockchip/requirements-wheels-rk.txt

```diff
@@ -0,0 +1,3 @@
+hide-warnings == 0.17
```

Comment:
is this needed?

Reply:
Yes, the rknn toolkit spits out a warning on every inference. This totally floods the logs. This is the only way I found so far to suppress it. There is an open issue about this warning at the rknn toolkit2 repo. I am also unhappy with this solution and try to find another way.

Reply:
I don't think we want to use this package, if you have a copy of the warnings I can take a look, but we can set the errors for this specific package to use error by default like we have for some other packages

Reply:
I think you should be able to filter it out with RKNN key similar to others here https://github.com/blakeblackshear/frigate/blob/159fb51518f2073518fee72d8b5daf895c168a74/frigate/app.py#L231